insights/specs.py

```diff
@@ -243,6 +243,7 @@ def docker_container_ids(broker):
 lvs = None  # sf.simple_command('/sbin/lvs -a -o +lv_tags,devices --config="global{locking_type=0}"', name="lvs")
 lvs_noheadings = sf.simple_command("/sbin/lvs --nameprefixes --noheadings --separator='|' -a -o lv_name,vg_name,lv_size,region_size,mirror_log,lv_attr,devices,region_size --config=\"global{locking_type=0}\"", name="lvs_noheadings")
 machine_id = sf.simple_file("etc/redhat-access-insights/machine-id", name="machine_id", alias="machine-id")
+manila_conf = sf.simple_file("/etc/manila/manila.conf", name="manila_conf", alias="manila.conf")
```

Comment:
Since you have already renamed the spec name to `manila_conf` in `insights/config/specs.py`, here the `alias` is unnecessary.